sites/docs/src/content/docs/tutorials/tests_and_test_data/components/03_project_setup.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "3. Repository Setup"
+subtitle: Configuring your nf-core pipeline repository for testing with nf-test
+weight: 30
+---
+
+## Example nf-core Repository Structure
+
+Here is an example of an nf-core pipeline project with nf-test:
+
+```
+my-pipeline/
+├── main.nf                          # Main pipeline script
+├── nextflow.config                  # Main pipeline configuration
+├── nextflow_schema.json            # Pipeline parameter schema
+├── nf-test.config                  # nf-test configuration
+├── modules.json                    # Module dependencies
+├── tests/                          # Pipeline-level tests
+│   ├── default.nf.test            # Main pipeline test
+│   ├── nextflow.config            # Test-specific config
+│   └── .nftignore                 # Files to ignore in snapshots
+├── conf/                           # Configuration profiles
+│   ├── test.config                # Test profile
+│   ├── test_full.config           # Full test profile
+│   ├── base.config                # Base configuration
+│   └── igenomes.config            # Reference genome configurations
+├── assets/                         # Pipeline assets
+│   ├── samplesheet.csv            # Test samplesheet
+│   └── schema_input.json          # Input validation schema
+├── modules/nf-core/               # Module tests
+│   └── */tests/main.nf.test      # Individual module tests
+├── subworkflows/nf-core/          # Subworkflow tests
+│   └── */tests/main.nf.test      # Individual subworkflow tests
+└── .nf-test/                      # nf-test working directory
+```
+
+### 1. Key Configuration Files
+
+#### `nf-test.config` (Main Configuration)
+
+**Purpose**: Controls nf-test global settings
```

Comment:
I would explain more of what these do, some are sort of important (e.g. profile means `test`  will be default across all tests), `triggers` and ignore might be important (and the latter should be ending in `/tests/*`)

Reply:
i provide a link to direct nf-test config docs below